src/components/flash/flash.js

```diff
@@ -223,19 +233,23 @@ class Flash extends React.Component {
    * @param {Object} ev
    * @return(Void)
    */
-  toggleDialog = (key, ev) => {
-    if (ev) { ev.preventDefault(); }
-
-    let state = this.state.dialogs[key];
-    // open/close the dialog
-    this.setState({ dialogs: { [key]: !state }});
-
-    // start/stop the timer if the dialog opens or closes
-    if (state) {
-      this.startTimeout();
-    } else {
-      this.stopTimeout();
-    }
+  toggleDialog = (key) => {
```

Comment:
Yeah its the nesting of the functions that im most curious about. Im trying to decide in my head which i prefer. I suppose there is no real difference except the function calls in the spec look off (comments below) but really that is because they are calling the functions directly rather than through re-rendering and clicks.